packages/react-generator/src/generator/page.js

```
@@ -376,34 +383,82 @@ const generateReactCode = ({ schema, name, type, componentsMap }) => {
+
+  console.log(statement, 'stateStatement>>>>>>')
+
+  // 生成模板
   const result = `${imports.join('\n')}
 
-  export default class ${name} extends React.Component {
-    ${stateStatement}
+  export default ${name} = () => {
+    ${statement}
 
     ${getters.join('\n')}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)